README.md

```diff
@@ -1,25 +1,77 @@
+
+    mv run_tests ~/catkin_ws/src/ros_exercises/
+
+Then make the binary executable with ```chmod```
+
+    chmod +x run_tests
+
+First, kill all running ROS processes. Then start ```roscore```.
+
+Finally, run the following in a new terminal to begin testing:
+
+     cd ~/catkin_ws/src/ros_exercises/
+    ./run_tests
+
+You should be graded on the completion of 6 tests. Be careful with naming the various files defined in the handout correctly or else you won't be graded properly.
+
+Additionally the `run_tests` file will generate a file called `log.npf` which you **must upload to gradescope for credit**.
+
+
+### Automatic grade Breakdown
+
+This table shows how the different tasks will be graded by the automated grader.
+| Task                    |      Grade        |
+|-------------------------|-------------------|
+| Simple Publisher        |      0.5 points   |
+| Simple Subscriber       |      0.5 Points   |
+| Complex Publisher       |      0.5 Points   |
+| Complex Subscriber      |      1.0 Points   |
+| Launch File             |      0.5 Points   |
+| **Total**               |   **3.0 Points**  |
+
+
+
+### Manual Grading Portion
+If you have successfully completed all parts, you should receive a score of 3.0/4.0 on Gradescope. The rest of your score (1.0 points) will be a manual grading of your package ``ros_exercises`` by the staff of your git repository, which should be avaliable on your `github.mit.edu` account. Submit a zip file of your repository to the Gradescope assignment **Lab 1C: Intro to ROS -- Git repo submission**
+
+### The Manual Grade Breakdown
+This table shows the grading breakdown of the manually graded portion of the lab.
+| Task                                | Grade          |
+|-------------------------------------|----------------|
+| Simple Publisher RQT Screenshot     | 0.1 Points     |
+| Simple Subscriber RQT screenshot    | 0.1 Points     |
+| Fake Scan RViz ScreenShot           | 0.1 Points     |
+| Open Space Publisher RQT Screenshot | 0.2 Points     |
+| Custom Message file inspection      | 0.2 Points     |
+| Launch file inspection              | 0.1 Points     |
+| Setting  ROS parameters	      | 0.2 Points     |
+| **Total**                           | **1.0 Points** |
+
 ## References
 
 The following are selected chapters from the ROS Wiki [Documentation](http://wiki.ros.org/) and [Tutorials](http://wiki.ros.org/ROS/Tutorials). If you understand all the concepts covered in these exercises, you should be ready for the following exercises and for most of the ROS related tasks you will be performing throughout the first few labs. For more on ROS, visit the [ROS Wiki](http://wiki.ros.org/) section and follow the links to learn more.
```

Comment:
"you should be ready for the following exercises" -> replace with "you will be ready for the exercises of this lab (given below)"